Resources/public/js/views/fields/ez-date-editview.js

```diff
@@ -103,12 +325,46 @@ YUI.add('ez-date-editview', function (Y) {
          * @return {Number}
          */
         _getFieldValue: function () {
-            var valueOfInput = this._getInputNode().get('valueAsNumber');
-            if (valueOfInput) {
-                return {timestamp: valueOfInput/1000};
-            }
-            return null;
+            var valueOfTextInput = Y.Date.parse(this._getInputNode().get('value'));
+
+                if (valueOfTextInput !== null){
+                    valueOfTextInput= Y.Date.format(valueOfTextInput, {format:"%s"});
+                    return {timestamp: Y.Number.parse(valueOfTextInput)};
+                }
+                return null;
+
         },
+    },{
+        ATTRS: {
+            /**
+             * Checks if browser supports HTML5 date input
+             *
+             * @readOnly
+             */
+            supportsDateInput: {
+                valueFn: '_detectInputDateSupport',
+                readOnly: true
+            },
+
+            /**
+             * YUI calendar instance
+             *
+             * @readOnly
+             */
+            calendar: {
+                readOnly: true
+            },
+
+            /**
+             * The valid date filled in the input to update the calendar
+             *
+             * @readOnly
```

Comment:
you should add the type of the value stored by this attribute (`@type` tag see http://yui.github.io/yuidoc/syntax/) as there are a lots of ways to represent a date.